An attempt to adjust a bucket quota failed.
Project: {{allocation.project.title}}
Resource/PI: {{allocation}}
Bucket: {{bucket_name}}
New Quota: {{quota}}
Allocation URL: {{ allocation_url }}

The following errors were recorded during processing:
=====================
Task: 
{{task.func}}

Task Exception: 
{{task.result|safe}}
=====================
